src/store/state/tombstone-model.ts

```diff
@@ -0,0 +1,6 @@
+import { TombStoneIF } from '@/interfaces'
+
+export const tombStoneModel: TombStoneIF = {
+  keycloakRoles: [],
```

Comment:
Is the intent for these to use the interfaces you declared above?

Reply:
Yes, you bet. This is just the initialization of the model. So when we bind this model to the component we can call all of the properties simply. See the StateExample component line 48.
No matter how many properties we have in this matter we can access them all in the component with the short cut binding from Vuex-class. `@State modelName: type`.    I'm sure they is many ways to do this, so i'm totally open to ideas!

Reply:
I'm wondering why the fields are initialized as empty arrays instead of using the interface types? I may be confused... might make more sense when I pull these changes and run it for myself :)